### ISSUE TITLE
answer: chipmusic sites

### ISSUE THREAD

**Issue by wothke** [flvi0/chipmusic-convert#19]
since you asked about other chipmusic sites...

my page: https://www.wothke.ch/playmod/

emulates and plays most of the songs from modland.com and vgmrips.net
online directly in the web browser 

cheers.

**Comment by drlight-code** [comment]
Cool, thanks! I must admit this project has been stalled since quite a while. Just out of curiosity do you, or did you use any of it?

**Comment by wothke** [comment]
no, I just came across your project by accident when searching for something else .. I would guess that you might be using some of the same underlying emulators for your recorder that I am using for my online player (see "Basics" tab on my page)
